app/src/HomePage.php

```diff
@@ -1,18 +1,26 @@
 <?php
 
+use SilverStripe\Core\Config\Config;
 use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
 
 class HomePage extends Page
 {
     private static $db = [
-        'RightContent' => 'HTMLText'
+        'RightHeading' => 'Varchar',
+        'RightContent' => 'HTMLText',
+        'RightCtaLabel' => 'Varchar',
+        'RightCtaLink' => 'Varchar',
     ];
 
     public function getCMSFields()
     {
         $fields = parent::getCMSFields();
-        $field = HTMLEditorField::create('RightContent', 'Right Content');
-        $fields->addFieldToTab('Root.RightContent', $field);
+        $db = HomePage::config()->get('db', Config::UNINHERITED);
+        foreach ($db as $fieldName => $type) {
```

Comment:
I don't like this.... but you ignored my previous change request for it and while this is technically not best practice it's also not doing any harm so I'll leave it.